### PR TITLE
update messaging about power needed for firmware updates

### PR DESCRIPTION
**Pull request by ahoneybun**
Having the laptop (darp5/6 and galp3-c/galp4) connected to power over USB Type-C while updating the firmware does not work so we need to let the user know this is the case.

### CODE REVIEW

**Comment by ahoneybun** [comment]
@leviport I added the spacing and trying to use textwrap.

File: gtk/src/dialogs/mod.rs [hunk 1]
```diff
@@ -1,3 +1,5 @@
+extern create textwrap;
```
Comment (leviport) [inline gtk/src/dialogs/mod.rs]: I believe this is what is causing the build to fail. Change `create` to `crate` and it should work better.

**Comment by ahoneybun** [comment]
@leviport fixed.

File: gtk/src/dialogs/mod.rs [hunk 2]
```diff
@@ -1,3 +1,5 @@
+extern crate textwrap;
+use textwrap:fill;
```
Comment (leviport) [inline gtk/src/dialogs/mod.rs]: You'll need another colon here. Should be `use textwrap::fill;`

**Comment by ahoneybun** [comment]
hidpi
lodpi
800x512

I tested the changes in a few different resolutions and on both lodpi and hidp as per @leviport  .

File: gtk/src/dialogs/mod.rs [hunk 3]
```diff
@@ -31,7 +32,8 @@ impl FirmwareUpdateDialog {
         let mut header = ["Firmware version ", version, " is available."].concat();
 
         if has_battery {
-            header.push_str(" <b>Plug into power</b> before you begin.");
+            header.push_str(" Connect your computer to power. <b>USB Type-C</b> charging is not supported for firmware updates.");
+	    header = (&*fill(&header, 75)).to_string();
```
Comment (jackpot51) [inline gtk/src/dialogs/mod.rs]: This line is not indented correctly.

Comment (ahoneybun) [reply gtk/src/dialogs/mod.rs]: Fixed in the latest commit (or should be).

File: gtk/src/dialogs/mod.rs [hunk 4]
```diff
@@ -31,7 +32,8 @@ impl FirmwareUpdateDialog {
         let mut header = ["Firmware version ", version, " is available."].concat();
 
         if has_battery {
-            header.push_str(" <b>Plug into power</b> before you begin.");
+            header.push_str(" Connect your computer to power. <b>USB Type-C</b> charging is not supported for firmware updates.");
+	        header = (&*fill(&header, 75)).to_string();
```
Comment (jackpot51) [inline gtk/src/dialogs/mod.rs]: This is exactly one space off now

Comment (ahoneybun) [reply gtk/src/dialogs/mod.rs]: It should look correct after commit:
https://github.com/pop-os/firmware-manager/pull/77/commits/e8494d07d75b9a01e42ac6295587888c092185fc

**Comment by jackpot51** [comment]
No, you did not fix indentation. I went ahead and fixed it and squashed this into one commit.

**Pull request merged**